charts/line-chart/src/LineChart.tsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { Chart, Line } from '@lg-charts/core';
+
+import { LineChartProps } from './LineChart.types';
+
+export function LineChart({ data, ...rest }: LineChartProps) {
```

Comment:
This probably shouldn't exist yet. It make more sense to just expose the composable pieces. This may be something to do at a later date.